src/Sarif.Converters.UnitTests/TestData/FortifyFprConverter/ExpectedOutputs/TwoResultsWithNodeRefs.sarif

```diff
@@ -269,7 +279,9 @@
                           "text": "Concat(1 : return)"
                         }
                       },
-                      "kind": "InOutCall"
+                      "kinds": [
+                        "InOutCall"
```

Comment:
>InOutCall [](start = 25, length = 9)

Now wait a minute -- seeing this I remember doing a similar review in the last day or so, where I suggested you use some of the "suggested" values, like
```
"kinds": [
  "exit",
  "scope"
]
```
So didn't you already do this change?

Reply:
Oh! This is the same PR :-) nm

---
In reply to: [258273748](https://github.com/Microsoft/sarif-sdk/pull/1292#discussion_r258273748) [](ancestors = 258273748)